data_structures/skip_list.cpp

```diff
@@ -0,0 +1,228 @@
+/*
```

Comment:
this block is still not up to the standards. Please check the above comments, references, links etc provided to you for your convenience.

Reply:
There are links, references, function documentations, sample codes provided. please check them.